chaincode-labs/chaincode-podcast/tracepoints-and-monitoring-the-bitcoin-network.md

```diff
@@ -85,317 +85,317 @@ Hope you enjoy it, too.
 
-Speaker 0: 00:06:48
+Adam Jonas: 00:06:48
 
 And so how do you like arrange something like that?
 You reach out to the various pools or just them and just say, I want to try this.
 
-Speaker 2: 00:06:54
+0xB10C: 00:06:54
 
 I reached out to two pools.
-One of them was F2 pool and they said, hey, let's go.
+One of them was F2 pool and they said, hey, let’s go.
 Yeah.
 
-Speaker 0: 00:06:58
+Adam Jonas: 00:06:58
 
 Okay, cool.
 
-Speaker 2: 00:06:58
+0xB10C: 00:06:58
 
 Yeah.
-And going back to the earlier story In this very night of taproot activation actually, the second block arrived from f2 pool this time and didn't include any taproot spans.
-Third block arrived, didn't include any taproot spans.
+And going back to the earlier story In this very night of taproot activation actually, the second block arrived from f2 pool this time and didn’t include any taproot spans.
+Third block arrived, didn’t include any taproot spans.
 So by the time the fourth block arrived, all these taproot spans were confirmed.
 And it later turned out that these pools had upgraded in time, burned for signaling, but the issue was that their peers were old and they had some weird manual peer configuration, which then caused problems for them.
-Their peers couldn't relay these pay-to-type-root spans because they are non-standard for them.
+Their peers couldn’t relay these pay-to-type-root spans because they are non-standard for them.
 
-Speaker 1: 00:07:34
+Mark Erhardt: 00:07:34
 
-So basically they were up to date and ready to go and actually correctly signaling, but just didn't see the Taproot transactions because their peers filtered them out and dropped them as nonstandard.
+So basically they were up to date and ready to go and actually correctly signaling, but just didn’t see the Taproot transactions because their peers filtered them out and dropped them as nonstandard.
 Correct.
 
-Speaker 2: 00:07:45
+0xB10C: 00:07:45
 
 Yeah.
 
-Speaker 0: 00:07:46
+Adam Jonas: 00:07:46
 
 So what was remediation for that?
 You chat with them, you figure it out.
 They change their peers and then off and running?
 
-Speaker 2: 00:07:53
+0xB10C: 00:07:53
 
 Yeah, they had some custom code and I think they dropped that.
 I think they now keep more closely to the Bitcoin core releases than before.
 And then after a few weeks, the AMP pool took a bit longer.
-We don't know exactly because the communication was a bit more difficult, but we know, they know my page attack would span just as all other pools are.
+We don’t know exactly because the communication was a bit more difficult, but we know, they know my page attack would span just as all other pools are.
 
-Speaker 0: 00:08:13
+Adam Jonas: 00:08:13
 
 Cool.
 And so, I mean, this is the service that you provide to the community.
 Correct.
```

Comment:
This "Correct" line here was mistakenly attributed to Jonas. We expect from reviewers to catch those